kazoo/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.8.0'
+__version__ = '2.9.0'
```

Comment:
Note my black PR will complain about the use of single quotes instead of double quotes. I'll rebase if need be. 

How do you feel about leveraging setuptool_scm to auto generate this file from git tag instead?

Reply:
Thanks, I will merge this one then, and let you rebase yours.

Love the idea about the use of `setuptool_scm`. I will try to implement it in another PR, unless you already have something ready on your end!